homeassistant/components/roomba/irobot_base.py

```diff
@@ -95,7 +95,7 @@ def device_info(self):
             "model": self._sku,
         }
 
-    def register_callback(self):
+    async def async_added_to_hass(self):
         """Register callback function."""
         self.vacuum.register_on_message_callback(self.on_message)
```

Comment:
Is this async safe? No blocking or I/O?

Reply:
Yes. It only adds the function to a list. It is defined [here](https://github.com/pschmitt/roombapy/blob/07d4965359035ad0024b45a63f8d9b5b3efa0dd8/roomba/roomba.py#L179).

Reply:
Good!